src/sentry/static/sentry/app/views/settings/organization/rateLimit/rateLimitView.jsx

```diff
@@ -56,6 +57,10 @@ class AccountLimit extends React.Component {
   }
 }
 
+const TextBlockStyled = styled(TextBlock)`
```

Comment:
this is a nice way of combining these styles, tho it still would be trouble if we ever wanted more than two (which maybe we don't so for now who cares)

Reply:
Yeah, this was more of solving the one off issue of the component's default 30px margin being too much. The jury is out on whether or not this is better than simply adding the styles inline.